flutter/rsvp/lib/Models/JsonHandler.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:rsvp/Models/event.dart';
+
+class JsonHandler {
+  Event eventObj;
+
+  FromEventListtoJsonList() {
```

Comment:
Capitalize "To" in `FromEventListtoJsonList` and the same for `FromJsonListtoEventList`